ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh

```diff
@@ -31,7 +31,8 @@ tempest init openshift
 
 pushd ~/tempest/openshift
 
-discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create identity.v3_endpoint_type public
+TEMPEST_CONF_OVERRIDES=${TEMPEST_CONF_OVERRIDES:-}
+discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create identity.v3_endpoint_type public ${TEMPEST_CONF_OVERRIDES}
 
```

Comment:
Is it possible to also copy the `tempest.log` that gets generated. The testr results rendered as HTML are useful, but in some cases, they won't have all the information to triage a failure

Reply:
(this comment is appropriate after line 94... but github won't let me post it there)

Reply:
and also the console log from running the tempest tests can be redirected and gathered